examples/xdebug-prestashop/docker-compose.yml

```
@@ -0,0 +1,38 @@
+name: flashlight-develop-prestashop
```

It may be preferable to use the same name as the folder
```suggestion
name: flashlight-xdebug-prestashop
```